CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changelog
+
+## v 03th May 2024
```

Comment:
This line can be probably be removed for now

Reply:
done